purchase_request/wizard/purchase_request_line_make_purchase_order.py

```diff
@@ -1,24 +1,22 @@
 # Copyright 2018-2019 Eficent Business and IT Consulting Services S.L.
-# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl-3.0).
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl-3.0).
 
 
 from datetime import datetime
 
 from odoo import _, api, fields, models
 from odoo.exceptions import UserError
 
-import odoo.addons.decimal_precision as dp
-
 
 class PurchaseRequestLineMakePurchaseOrder(models.TransientModel):
     _name = "purchase.request.line.make.purchase.order"
     _description = "Purchase Request Line Make Purchase Order"
 
     supplier_id = fields.Many2one(
-        "res.partner",
+        comodel_name="res.partner",
         string="Supplier",
         required=True,
-        domain=[("supplier", "=", True), ("is_company", "=", True)],
```

Comment:
Because in Odoo 13 has no longer field "supplier" and "customer" anymore.

ref : https://github.com/odoo/odoo/commit/8626257bc7b9ee772006eb17d12f56344e594f25

Reply:
Thanks for the explanation :)
I would like to fix allocations and port them together to v13.
I opened a PR https://github.com/OCA/purchase-workflow/pull/791, could you review it?

Reply:
Sure, I can.

Reply:
Migrations script are used to fit data from previous versions to a new data model defined in the file name version, for example:
On v11 purchase_request didn't use allocations, to fill up allocations in v12 from purchase requests that are already in the system, migration script was required.
Migrations script have 2 stages that are defined in different files:
- pre-migration.py are ORM/SQL scripts that fit old data model in the new one. for example if a field name is changed, you must rename column from database before do the update.
- post-migration.py are executed after module updates.
I would be interested that you look also on *_hooks.py  files.

I would recommend to come back migrations script that I asked to remove previously.
I think we won't need migrations script from v12 to v13 because data model hasn't been altered.

Reply:
@kongrattapong 
Maybe better to replace it by context:
`{'res_partner_search_mode': 'supplier', 'default_is_company': True}`

Reply:
@hveficent sorry for late reply and thank you for explain.

@CasVissers thank for your review.